### DISCUSSION TITLE
Installed Apps creds view in UI [Oct 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3999] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **prasanth**  
#### yeah but if i invite a user to my project his role applies to the project i invited but when he creates new project the creds are visible to him since he is owner  
#  
> 💁 **o1lab**  
#### fair point  
#  
> 💁 **o1lab**  
#### we are planning to move some apps at root level which could be accessed by owners  
#  
> 🙋‍♂️ **prasanth**  
#### can we restrict users from creating projects <@310157751395287040> ?   
####    
#### this seems to be already fixed but its not working as mentioned in this issue - https://github.com/nocodb/nocodb/issues/793   
####    
#### i would like the users i invited to access the projects only they were invited and other projects should not be visible to them and only super admin can give a user access to create projects   
####    
#### is this feasibility currently exists?  
#  
> 🙋‍♂️ **prasanth**  
#### i would be glad if someone can guide me with this.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1022142045579640842)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
